src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -258,14 +258,11 @@ func (c controller) PostInstallConfigs(wg *sync.WaitGroup, status *ControllerSta
 		break
 	}
 
-	errMessage := ""
 	err := c.postInstallConfigs()
 	if err != nil {
-		errMessage = err.Error()
 		status.Error()
+		c.log.WithError(err).Error("Failed to finish post install configurations")
 	}
-	success := err == nil
-	c.sendCompleteInstallation(success, errMessage)
```

Comment:
If it's no longer required just replace it with a log:
c.log.Infof("Installation completed")